server.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Make sure this is the latest server.xml from the 9.0.50 install, then make appropriate changes.  The question is: What did you change from OOTB?

Reply:
Yes it is latest server.xml. I actually copied with `docker cp` server.xml to local working machine and made changes. The changes that were added are turning off SSL engine and removing the accesslog directive
```xml
<Valve className="org.apache.catalina.valves.AccessLogValve" directory="logs"
               prefix="localhost_access_log" suffix=".txt"
               pattern="%h %l %u %t &quot;%r&quot; %s %b" />
```

```xml
<Listener className="org.apache.catalina.core.AprLifecycleListener" SSLEngine="off" />
```